app/backend/utils.js

```diff
@@ -1,3 +1,36 @@
+const IBAN = require('iban');
+
 module.exports.checkPasswordLength = function (password, response){
   return (password.length < 6)
-}
\ No newline at end of file
+}
+
+module.exports.checkIBAN = function(iban){
+  return IBAN.isValid(iban)
+}
+
+module.exports.checkTCKN = function(value) {
+  value = value.toString();
```

Comment:
One small comment here describing for what purpose the function is created would be useful for our further selves.